src/transformers/modeling_utils.py

```diff
@@ -345,6 +346,20 @@ def get_output_embeddings(self):
         """
         return None  # Overwrite for models with output embeddings
 
+    def get_layers(self):
+        """
+        Returns the model's transformer layers.
+
+        Returns:
+            :obj:`List` or :obj:`nn.ModuleList`:
+                A list or :obj:`nn.ModuleList` containing all transformer layers.
+        """
+        base_model = getattr(self, self.base_model_prefix, self)
```

Comment:
Really hope that this works for all models! The only model where I see that it doesn't work at the moment is T5, but maybe we can just overwrite it there?

Reply:
I think overwriting for the few models that need it is fine.